graphql/client.go

```diff
@@ -40,9 +40,8 @@ type Client interface {
 	// (Errors are returned.) But again, MakeRequest may customize this.
```

Comment:
The documentation here will need some updating!  Mostly you can move the descriptions of `query` etc. down into `Payload`, and `retval` down into `Response`, and just replace them with something brief about request and response.